osu.Game/Graphics/DrawableDate.cs

```diff
@@ -58,8 +58,10 @@ private void updateTimeWithReschedule()
 
         public override bool HandleMouseInput => true;
 
-        private void updateTime() => Text = date.Humanize();
+        protected virtual string Format() => Text = date.Humanize();
 
-        public string TooltipText => date.ToString();
+        private void updateTime() => Format();
+
+        public virtual string TooltipText => string.Format($"{date:d MMMM yyyy H:mm \"UTC\"z}");
```

Comment:
Format should be `MMMM d, yyyy h:mm tt \"UTC\"z` to match web.

![image](https://user-images.githubusercontent.com/6215387/40884562-37928d8e-66ec-11e8-89cf-807a79f71202.png)